* Fixes #58 - provides SINGLE_LANGUAGE option when configuring

BrianSidebotham opened the pull request:
Adds a single language option. Safely builds a language, even partially translated languages such as `fr`, only what is translated is included in the build.

To use this commit, configure a single language build with:

    cmake -DSINGLE_LANGUAGE=en

Available languages so far are `en|fr|it|pl`

The package name is transformed when using **SINGLE_LANGUAGE** so that the language is included in the package tarball.

kicadjenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci.kicad-pcb.org/job/kicad-doc-testing-pull-request/21/
Test FAILed.

BrianSidebotham commented:
I've fixed #60 which meant you couldn't build `en` as the **SINGLE_LANGUAGE** option. I've also added a new configuration option **BUILD_FORMATS** which is a list of build formats to build. Use it to specify a list of build formats (as we only have two formats, this option is simply for selecting only one of them.)

`-DBUILD_FORMATS=html` or `-DBUILD_FORMATS=pdf`

BrianSidebotham commented:
If the ci server fails again after the new commits, please paste the output somewhere. I don't think I have credentials for the ci.

BrianSidebotham commented:
Oh, okay I see it was just a Java face plant. I'll wait for the next job to complete.

kicadjenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci.kicad-pcb.org/job/kicad-doc-testing-pull-request/22/
Test PASSed.

kicadjenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci.kicad-pcb.org/job/kicad-doc-testing-pull-request/23/
Test PASSed.

kicadjenkins commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci.kicad-pcb.org/job/kicad-doc-testing-pull-request/24/
Test PASSed.
